manifest/manifest.py

```diff
@@ -106,6 +107,7 @@ def run(
         prompt: Union[Prompt, str],
         input: Optional[Any] = None,
         gold_choices: Optional[List[str]] = None,
+        force_rerun: bool = False,
```

Comment:
So what does this do in comparison to `overwrite_cache`? Is there some reason you want to add an id?

Reply:
Adding the `run_id` makes the cache key different, and therefore the results are written to the cache. This way you can run the same prompt+settings many times and cache all of the results.

Compare to `overwrite_cache`, where you can do many runs, but only the latest run is cached.

Reply:
Perhaps the parameter should be named something different, like `cache_reruns`